### ISSUE TITLE
Missing labels

### ISSUE THREAD

**Issue by jakubsadel** [aysim/dynnet#3]
Hello, I am quite a beginner in the plot of satelite imaginery and I have a question about the different number of labs in this repository, paper and on the actual disk.

In your repository in the train, val, test split files the sum of all the labels is 1752. While in the paper it says that there are 1800. Ok, this is not a big difference I am able to understand it. What I don't understand is that in the labels folder that is downloadable on disk there are only 1320 annotated tif files. 

What happened that there is such a big difference? Do I need to process these files additionally somehow, or simply the datset currently available to the disk is incomplete compared to the one you ran your code on?

Regards
Jakub Sądel

**Comment by aysim** [comment]
Hi, sorry for the confusion. The sum of the train, val, and test split files is 1752 because we had two cubes missing when we were doing the experiments. In the readme you can find:

> During the experiments, 2 cubes from the training set were not available so we exclude them. However now they are ready to use, please modify the train.txt accordingly. Specifically, the id's of the missing cubes: 1417_3281_13 and 2235_3403_13.

Therefore we had 1272 images for training (53*24), 240 for validation (10*24), and 240 for testing. The missing 2 cubes were completed by the publishing deadline. So 1800 in the paper is accurate. On the other hand, we only open source the training annotations (55 cubes --> 55*24=1320 in total). We do not provide the annotations for the validation and test cubes. To validate your models on our validation and test set you can use our codalab.

**Comment by jakubsadel** [comment]
thanks for your response and clarifying
